index.html

```diff
@@ -2187,6 +2187,15 @@ <h5>Character naming template</h5>
 &lt;span class="uname">U+XXXX Unicode_character_name&lt;/span>&lt;/span>
 </pre>
 
+<p>For combining characters (such as <span class="codepoint" translate="no"><bdi lang="en">&#x25cc;&#x0300;</bdi><span class="uname">U+0300 COMBINING ACCENT GRAVE</span></span>), if the specification uses a font which supports display with a <span class="codepoint" translate="no"><bdi lang="en">&#x25cc;</bdi><span class="uname">U+25CC DOTTED CIRCLE</span></span>, then a dotted circle should be used before the combining character:</p>
```

Comment:
This is ambiguous? Does it mean "... uses a font which supports the dotted circle character with combining marks..."? How does a specification author know this? I think this is getting too complex and would rather not having folks flinging 25CCs about.

I think we should discuss automation of using an image? Or whether we can live with the combining mark showing without dotted circles?